django/contrib/admin/static/admin/css/base.css

```diff
@@ -626,22 +630,53 @@ ul.messagelist li {
     display: block;
     font-weight: 400;
     font-size: 0.8125rem;
-    padding: 10px 10px 10px 65px;
-    margin: 0 0 10px 0;
-    background: var(--message-success-bg) url(../img/icon-yes.svg) 40px 12px no-repeat;
-    background-size: 16px auto;
+    padding: 0.625rem 0.625rem 0.625rem 4.063rem;
+    margin: 0 0 0.625rem 0;
+    background-color: var(--message-success-bg);
     color: var(--body-fg);
     word-break: break-word;
+    position: relative;
+}
+
+ul.messagelist li::before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 0.625rem;
+    transform: translateY(-50%);
+    width: 1rem;
+    height: 1rem;
+    mask-image: url(../img/icon-yes.svg);
+    -webkit-mask-image: url(../img/icon-yes.svg);
+    mask-size: contain;
+    -webkit-mask-size: contain;
+    mask-repeat: no-repeat;
+    -webkit-mask-repeat: no-repeat;
+    background-color: var(--icon-success-color);
 }
 
 ul.messagelist li.warning {
-    background: var(--message-warning-bg) url(../img/icon-alert.svg) 40px 14px no-repeat;
-    background-size: 14px auto;
+    background-color: var(--message-warning-bg);
+}
+
+ul.messagelist li.warning::before {
+    mask-image: url(../img/icon-alert.svg);
+    -webkit-mask-image: url(../img/icon-alert.svg);
+    width: 0.875rem;
+    height: 0.875rem;
```

Comment:
So we need this because the icon’s visuals in the SVG are drawn closer to the bounding box? To me it’s an improvement if the icon’s width and height was `1rem`, so I’d recommend skipping that override.

Reply:
Will do.